tests/impact/client/helpers.py

```diff
@@ -157,11 +157,14 @@ class IDs:
     FMU_SECONDARY = 'test_filter_fmu_id'
     EXPERIMENT_PRIMARY = 'pid_20090615_134'
     EXPERIMENT_SECONDARY = 'filter_20090615_135'
+    CASE_PRIMARY = 'case_1'
```

Comment:
Can you factor out introduction of constants and corresponding renames across tests into a separate "chore" commit? Mixing that with the new API makes it difficult to separate feature from test improvement.

Reply:
Ah sorry for that. I will do that before merging.